### DISCUSSION TITLE
part-ii/2021-122/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#385] [Question comments]
Part II 2021 Probability and Measure: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2021-122/

**Comment by ArthurConmy** [comment]
<details>
<summary>Spoiler</summary>

The same functions

$$ \text{min} \left( n , \frac{1}{2^n} \lfloor 2^n f(x) \rfloor \right) $$

work for all parts, as for the second part this follows from monotone convergence, and for the final part note that eventually the $\text{min}(n, ...)$ will have no effect and hence the convergence will be uniform.

</details>
